fhir-mapping/src/test/ml-modules/root/test/suites/testUtils.sjs

```diff
@@ -0,0 +1,241 @@
+const test = require('/test/test-helper.xqy');
+
+const logger = require('./testLogger.sjs').instance('testUtils.sjs');
+
+const basePath = test.__CALLER_FILE__.split('/').slice(0, -1).concat('test-data').join('/');
+
+/**
```

Comment:
Are these comments left over from deleted functions?

Reply:
These comments are documenting argument types used below. I put them up here for lack of a better place to put them.